examples/api/segments/example1-callbacks.js

```diff
@@ -5,12 +5,53 @@
 
 'use strict'
 
-var newrelic = require('newrelic')
+const newrelic = require('newrelic')
+
+/**
+ * We'll stub out an async task that runs as part of monitoring a segment
+ *
+ * @param {Function} callback
+ */
+function myAsyncTask(callback) {
+  const sleep = new Promise((resolve) => {
+    setTimeout(resolve, 1)
+  })
+  sleep.then(() => {
+    callback(null, 'hello world')
+  })
+}
+
+// then we stub out the task that handles that task's result,
+// to show how the result is passed throughthe segment handler
+/**
+ * @param {string} greetings
+ * @param {Function} callback
+ */
```

Comment:
I feel like the jsdoc on all these example files is just getting in the way of showing stuff to the user but we can land and then later remove if needed.

Reply:
Yeah, I understand that perspective. I could take it or leave it. It's sort of unusual to have JSDoc comments in examples but more clarity is probably better than less, generally speaking.

Reply:
More clarity is good when theres clarity being added. Calling out that "greetings" and "callback" are parameters for generically meeting jsdoc's doesn't really add any clarity. I can already read that in the function. I think they are just noise getting in the way of understanding what we are trying to communicate.

That being said, I don't feel strongly enough that it needs to be handled via this PR.

Reply:
No problem. I think you're right on the money. It's no trouble to replace these blocks with `eslint-disable-next-line` 👍